plugins/Feedback/templates/index.twig

```diff
@@ -8,6 +8,24 @@
 {% block content %}
 
     <div id="feedback-faq" class="admin">
+        <div piwik-content-block content-title="{{ "Feedback_HowCanWeHelp"|translate|e('html_attr') }}" class="feedback-help-resources">
```

Comment:
I reckon be good to move this section under "About Matomo". 

We then need to see how to further optimise it as there will be three different help sections...  Community help also partially repeats the guides and FAQs. People might not notice the community help and professional help section when we show the "how can we help" section... 
![image](https://user-images.githubusercontent.com/273120/64654318-0025f980-d47d-11e9-89b5-f2b33b7a8069.png)

@katebutler 
could you merge the three sections into one section similar to this?
![image](https://user-images.githubusercontent.com/273120/64654426-55faa180-d47d-11e9-94b2-97c04db977ef.png)